src/core/PlayAreaApi.ttslua

```diff
@@ -77,5 +85,13 @@ do
     return getObjectFromGUID(PLAY_AREA_GUID).call("isInPlayArea", object)
   end
 
+  PlayAreaApi.getSurface = function()
+    return getObjectFromGUID(PLAY_AREA_GUID).getCustomObject().image
+  end
+
+  PlayAreaApi.updateSurface = function(url)
+    return getObjectFromGUID(IMAGE_SWAPPER).call("updateSurface", url)
+  end
```

Comment:
Turn the image updating into a single function

Reply:
Agreed, but will be handled in the API Updates PR